Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -774,7 +778,7 @@
   - CORPSES
   - DEODORANT AND A BATH
   - ENOUGH CABBAGES
-  - FIVE HUNDRED AND NINETY-NINE US DOLLARS
+  - FIVE HUNDRED AND NINETY-NINE PIECES OF EIGHT
```

Comment:
This one can go either way; it's specifically a reference to the PS3's launch price meme, which rocked the internet back in... 2005/6. So nineteen years ago. Probably best to just remove the line.